website/docs/getting-started/setup/upgrade-to-business-edition/docker.mdx

```diff
@@ -0,0 +1,129 @@
+---
+description: Follow the steps to upgrade the Appsmith community docker installation to a Business Edition.
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Docker
+
+Follow the steps listed below to **upgrade** your Community Edition (CE) to a Business Edition (EE) installed on docker.
+
+### Step 1: Database backup
+
+The Community and Business editions use Mongo DB, so you don't have to migrate the database. It's still recommended to [backup the Community Edition Mongo DB](/getting-started/setup/instance-management/appsmithctl#export-database).
+
+> Ensure that you backup the database with the docker instance **up** and **running**.
+
+### Step 2: Add or replace license key
+
+Go to the directory `~stacks/configuration` and open the `docker.env` file. Add/replace the Business Edition license key and save the file.
+
+```bash
+APPSMITH_LICENSE_KEY=<ENTER_THE_LICENSE_KEY>
+```
+
+### Step 3: Update image
+Go to the directory where the `docker-compose.yml` file is located. Open the `docker-compose.yml` file, and look for the `image:` key. Update the image in the `docker-compose.yml` file from Community Edition (_appsmith-ce_) to Business Edition (_appsmith-ee_). 
+
+The updated file should have the image attribute set as below:
+
+ ``` yaml
+  services:
+    appsmith:
+    #highlight-next-line
+      image: index.docker.io/appsmith/appsmith-ee
+```
+And save the file.
+
+> If you are using `docker run`, head to [recreate the instance](#step-4-recreate-the-instance) step.
+
+### Step 4: Recreate the instance
+
+<Tabs groupId="dockercommands">
+ <TabItem label="Docker Compose" value="using-docker-compose-command">
+ You can recreate the instance by running the below command:
+
+```bash
+docker-compose up -d
+```
+:::info replace license key
```

Comment:
Let this be a regular Note callout. Else when people see 'Replace license key' with an exclamation mark, they will infer this is a required step.

Also, why is this information being placed here? Maybe this should be mentioned in the place where they replace the license key in step 2?